icicle/tests/polynomial_test.cu

```diff
@@ -905,6 +968,21 @@ TEST_F(PolynomialTest, Groth16)
   // groth16_example.verify(proof); // cannot implement without pairing
 }
 
+
+TEST_F(PolynomialTest, DummyGroth16)
+{
+  // (1) construct R1CS and QAP for circuit with N inputs
+  Groth16Example<scalar_t, affine_t, projective_t, dummy_g2_t, dummy_g2_t> groth16_example(30 /*=N*/);
+
+  // (2) compute witness: randomize inputs and compute other entries [1,out,...N inputs..., ... intermediate values...]
+  auto witness = groth16_example.random_witness_inputs();
+  groth16_example.compute_witness(witness);
+
+  groth16_example.setup();
+  auto proof = groth16_example.prove(witness);
+  assert(groth16_example.dummy_verify(proof, witness));
```

Comment:
use gtest ASSERT_EQ instead. Note that assert() is not executing in release mode.